### ISSUE TITLE
MuseSounds Stuck Notes on Export

### ISSUE THREAD

**Issue by hostrauser** [musescore/MuseScore#17849]
### Issue type

Muse Sounds bug

### Bug description

MuseScore 4.0.2 on Windows 10. Score looks just fine, playback inside of MuseScore is perfect, but when I export to WAV or MP3 there are all sorts of stuck notes that don't drop when they should. It's not an orphaned tie, because there are no ties for several hundred measures (it's very staccato). And, like I said, it plays perfectly inside of MuseScore. 

After testing, determined this was a MuseSounds issue. A friend was able to export my score to audio using the default soundfont with no issue.

### Steps to reproduce

1. Choose a file with orchestral instruments.
2. Export to an audio format.
3. The created audio file will have stuck notes that don't drop/clear properly. The playback of the audio within MuseScore will be fine, however.

### MuseScore Version

4.0.2

### Regression

I don't know

### Operating system

Windows 10

**Comment by MarcSabatella** [comment]
Please attach the score so it can be investigated (zip it or rename to end in .zip so GitHub will accept it).

**Comment by hostrauser** [comment]
[Symphony N4 M1.zip](https://github.com/musescore/MuseScore/files/11647717/Symphony.N4.M1.zip)

Thank you, file is attached.

**Comment by bkunda** [comment]
@hostrauser is this also an issue with the latest master (4.1)?

**Comment by DmitryArefiev** [comment]
Tested on **4.0.2**, MuseSounds Windows10

@hostrauser  Here is my mp3 from the attached score
[Symphony N4 M1.zip](https://github.com/musescore/MuseScore/files/11679400/Symphony.N4.M1.zip)

Can you point where to find stuck the notes please?

Also, can you check in MuseHub if your MuseSounds are updated?

**Comment by hostrauser** [comment]
> Tested on **4.0.2**, MuseSounds Windows10
> 
> @hostrauser Here is my mp3 from the attached score [Symphony N4 M1.zip](https://github.com/musescore/MuseScore/files/11679400/Symphony.N4.M1.zip)
> 
> Can you point where to find stuck the notes please?
> 
> Also, can you check in MuseHub if your MuseSounds are updated?

Nope, your export is flawless. No stuck notes. Now I'm even more confused. I could never get that good of an export, and I don't know why.

All of my MuseSounds have a check mark under them in MuseHub, I assume that means they are current? If not, what should I be looking at.

**Comment by hostrauser** [comment]
> @hostrauser is this also an issue with the latest master (4.1)?

I'm afraid I don't understand. Forgive me, I'm very new to MuseScore, have only been using it about a month. When I check for update in MuseScore it says I am up to date, and the version upon load says 4.0.2.

**Comment by hostrauser** [comment]
So, WAV files are too big to attach, but I tried messing with sample sizes. At a rate of 32000 Hz, the WAV file had numerous dropped/missing notes on the export. At 48000 Hz, the stuck notes problem appears.

Here is what I get when I exported to MP3 (sample rate 48000 Hz, bit rate 128 kBits/sec)
[Symphony N4 M1 48000 128k.zip](https://github.com/musescore/MuseScore/files/11683566/Symphony.N4.M1.48000.128k.zip)

Just in the first two minutes:
0.07 stuck strings, brief
0.30 missed timpani note
0.57 missed strings
1.01 missed strings
1.38 stuck strings
1.47 stuck woodwinds, long

**Comment by DmitryArefiev** [comment]
@hostrauser Very interesting.. Now I able to reproduce it when Export MP3/WAV  at **48kHz**

Thanks for investigation!

**Comment by DmitryArefiev** [comment]
@bkunda This also occurs in master 4.1 (MuseSounds)
